lib/services/doctor-service.ts

```diff
@@ -42,15 +42,15 @@ class DoctorService implements IDoctorService {
 		if (hasWarnings) {
 			this.$logger.info("There seem to be issues with your configuration.");
 			await this.promptForHelp();
+		} else {
+			this.$logger.out("No issues were detected.".bold);
```

Comment:
will we print this on each `tns build ...` when there are no issues?

Reply:
No. This will be printed only on doctor command